packages/create-waku/src/index.ts

```diff
@@ -202,15 +203,37 @@ async function init() {
     await installTemplate(root, packageName, templateRoot, templateName);
   }
 
-  // TODO automatically installing dependencies
   // 1. check packageManager
   // 2. and then install dependencies
-
   console.log(`\nDone. Now run:\n`);
   console.log(`${bold(green(`cd ${targetDir}`))}`);
   console.log(`${bold(green(commands.install))}`);
   console.log(`${bold(green(commands.dev))}`);
   console.log();
+  // console.log(`Installing dependencies by running ${commands.install}...`);
+
+  // process.chdir(targetDir);
```

Comment:
My guess is that this makes it fail.
```suggestion
  process.chdir(targetDir);
```